ibis/expr/types/relations.py

```diff
@@ -2947,17 +2947,17 @@ def join(
         rated a movie:
 
         >>> tags.join(ratings, ["userId", "movieId"]).head(5)
-        ┏━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━┓
-        ┃ userId ┃ movieId ┃ tag            ┃ timestamp  ┃ rating  ┃
-        ┡━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━┩
-        │ int64  │ int64   │ string         │ int64      │ float64 │
-        ├────────┼─────────┼────────────────┼────────────┼─────────┤
-        │     62 │       2 │ Robin Williams │ 1528843907 │     4.0 │
-        │     62 │     110 │ sword fight    │ 1528152535 │     4.5 │
-        │     62 │     410 │ gothic         │ 1525636609 │     4.5 │
-        │     62 │    2023 │ mafia          │ 1525636733 │     5.0 │
-        │     62 │    2124 │ quirky         │ 1525636846 │     5.0 │
-        └────────┴─────────┴────────────────┴────────────┴─────────┘
+        ┏━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━┓
```

Comment:
Do you know why this resultset has changed?

Reply:
No, but I don't think it matters too much, duckdb could produce two different row orders for the same query if the query doesn't contain an order by.